File: 백준/Bronze/16431. 베시와 데이지/베시와 데이지.cc
#include <stdio.h>

int main(){
    int Bx, By, Dx, Dy,Zx, Zy, Bt, Dt;
    scanf("%d %d\n%d %d\n%d %d", &Bx, &By, &Dx, &Dy, &Zx, &Zy);
    Bx = Zx >= Bx ? Zx - Bx : Bx - Zx; 
    By = Zy >= By ? Zy - By : By - Zy;
    Bt = Bx >= By ? Bx : By;
    Dx = Zx >= Dx ? Zx - Dx : Dx - Zx; 
    Dy = Zy >= Dy ? Zy - Dy : Dy - Zy;
    Dt = Dx + Dy; 
    if(Bt < Dt){
        printf("bessie");
    }else if(Bt == Dt){
        printf("tie");
    }else{
        printf("daisy");
    }
}

File: 백준/Bronze/16431. 베시와 데이지/베시와 데이지.py
Bx, By = map(int, input().split())
Dx, Dy = map(int, input().split())
Zx, Zy = map(int, input().split())
Bx = Zx - Bx if Zx >= Bx else Bx - Zx; 
By = Zy - By if Zy >= By else By - Zy;
Bt = Bx if Bx >= By else By;
Dx = Zx - Dx if Zx >= Dx else Dx - Zx; 
Dy = Zy - Dy if Zy >= Dy else Dy - Zy;
Dt = Dx + Dy; 
if Bt < Dt:
    print("bessie")
elif Bt == Dt:
    print("tie");
else:
    print("daisy");